metadata-service/factories/src/main/resources/application.yml

```diff
@@ -28,15 +28,14 @@ authentication:
 
 # Authorization-related configurations.
 authorization:
-
   # Configurations for the default DataHub policies-based authorizer.
   defaultAuthorizer:
     enabled: ${AUTH_POLICIES_ENABLED:true}
     cacheRefreshIntervalSecs: ${POLICY_CACHE_REFRESH_INTERVAL_SECONDS:120}
 
 ingestion:
   enabled: ${UI_INGESTION_ENABLED:true}
-  defaultCliVersion: '${UI_INGESTION_DEFAULT_CLI_VERSION:@cliMajorVersion@}'
+  defaultCliVersion: "${UI_INGESTION_DEFAULT_CLI_VERSION:@cliMajorVersion@}"
```

Comment:
Is this change intended?

Reply:
Automatic reformatting to be consistent with the rest of the file